static/js/tools/timeline_bulk_download.ts

```diff
@@ -75,13 +76,18 @@ function downloadBulkData(
 }
 
 // TODO(beets): Define interfaces for all responses
-function saveToCsv(placeDcids: string[], statVars: string[], data: any): void {
+function saveToCsv(
+  placeDcids,
+  placeNames: { [dcid: string]: string },
+  statVars: string[],
+  data: any
+): void {
   const results = [];
-  results.push("placeDcid," + statVars.join(","));
+  results.push("placeDcid,placeName," + statVars.join(","));
   for (const place of placeDcids.sort()) {
-    const row = [place];
+    const row = [`"${place}"`, `"${placeNames[place]}"`];
     for (const sv of statVars) {
-      row.push(data[sv]["stat"][place]["value"]);
+      row.push(data[sv]["stat"][place]["value"] || "-");
```

Comment:
Is an empty CSV cell value better than "-" I wonder? (e.g, utils like pandas have built-ins to skip empty values)

Reply:
good point, i can leave it blank.